lib/peep.rb

```diff
@@ -1,6 +1,11 @@
 require 'pg'
 
 class Peep
+  attr_reader :post
```

Comment:
can you explain why you need an attr_reader here?

Reply:
That was to allow a test to have access to that particular method, it has since been updated though!